tensorflow/lite/delegates/nnapi/java/src/main/java/org/tensorflow/lite/nnapi/NnApiDelegate.java

```diff
@@ -29,7 +29,9 @@
   public static final class Options {
     public Options() {}
 
-    /** undefined */
+    /** undefined, specifies default behavior.
```

Comment:
Nit: This needs a line break, fix to something like:

/** 
 * undefined, specifies default behavior.
 * So far, the default setting of NNAPI is EXECUTION_PREFERENCE_FAST_SINGLE_ANSWER.
  */